app/routes/dashboard/purchase/confirm.tsx

```diff
@@ -0,0 +1,86 @@
+import { Outlet, redirect } from 'remix'
```

Comment:
Please remove all unused variables as indicated by the ESLint warnings below.
```suggestion
import { Outlet } from 'remix'
```